TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/200_Assets/Scripts/OutputTextureFeature.cs

```diff
@@ -0,0 +1,97 @@
+using UnityEngine;
```

Comment:
This Renderer Feature is used to display the Depth, Normals or SSAO texture in the new test scenes. 
It is only in the projects, not part of public URP codebase.